docs/sql/refresh.txt

```diff
@@ -20,4 +23,62 @@ of a table can be changed with the table parameter ``refresh_interval`` (see
 :ref:`sql_ref_refresh_interval`).
 
 
+Multiple Table Refresh
+======================
+
+.. Hidden: CREATE TABLE::
+
+    cr> CREATE TABLE IF NOT EXISTS parted_table (
+    ...   id long,
+    ...   title string,
+    ...   content string,
+    ...   width double,
+    ...   day timestamp
+    ... ) CLUSTERED BY (title) INTO 4 SHARDS PARTITIONED BY (day);
+    CREATE OK (... sec)
+
+.. Hidden: INSERT INTO::
+
+    cr> INSERT INTO parted_table (id, title, width, day)
+    ... VALUES (1, 'Don''t Panic', 19.5, '2014-04-08');
+    INSERT OK, 1 row affected (... sec)
+
+If needed, multiple tables can be defined comma-separated in a single 
+SQL request. This ensures that they all get refreshed and so their datasets 
+get consistent. The result message is printed if the request on every given 
+table is completed.
+
+::
+
+    cr> REFRESH TABLE locations, parted_table
+    REFRESH OK (... sec)
+
+
+Partitiones
+===========
+
+Additionally it is possible to define a specific ``PARTITION`` of the table 
```

Comment:
define a specific ``PARTITION`` of  a partitioned table

link partiioned, leave the explanation out